tidepods/cli.py

```diff
@@ -36,50 +36,28 @@ def cli():
-@click.option('-p', '--pfs', type=click.Path(dir_okay=False), required=True,
-              help='Path to output pfs file e.g. C:/tides/output.pfs')
-def pfs(**kwargs):
-    """Create pfs file for input to MIKE
-
-    Example use:
-
-    tidepods pfs -i C:/tides/aoi.shp -t C:/tides/template.txt -d 20170131
-    -o C:/tides/pts.shp -p C:/tides/outpfs.pfs
-    """
-    from . import pfs_shp as ps
-    ps.write_pfs(**kwargs)
-    ps.write_pts(**kwargs)
-
-
-@cli.command()
-@click.option('-i', '--infile', type=click.Path(dir_okay=False, exists=True), required=True,
-              help='Path to dfs0 file created using tidepods pfs e.g. C:/tides/aoi.dfs0')
+              help='Path to AOI raster file for points creation e.g. C:/tides/aoi.tif')
 @click.option('-d', '--date', type=DATEIN, required=True,
               help='Image acquisiton date (yyyymmdd) e.g. 20150131')
 @click.option('-t', '--timestamp', type=TIMEIN, required=True,
-              help='Image acquisition time (HH:MM) e.g. 12:34')
-@click.option('-s', '--shapefile', type=click.Path(dir_okay=False, exists=True), required=True,
-              help='Path to input pts shapefile created using tidepods pfs e.g. C:/tides/pts.shp')
+              help='Image acquisition time (HH:MM) e.g. 10:30')
 @click.option('-o', '--outfile', type=click.Path(dir_okay=False), required=True,
-              help='Path to output points shapefile containing the tidal values e.g. C:/tides/pts_tides.shp')
-@click.option('-p', '--sdkpath', type=click.Path(dir_okay=True, file_okay=False),
-              default=r'C:\Program Files (x86)\DHI\2016\MIKE SDK\bin', show_default=True,
-              help='Path to DHI SDK install directory')
-def tides(date, timestamp, **kwargs):
-    """Extract tide values from dfs0 file to a point shapefile
+              help='Path to output points shapefile containing the tidal values '
+              'e.g. C:/tides/pts_tides.shp')
+@click.option('-l', '--level', type=click.Choice(['LAT', 'MSL']), required=True,
+              help='Tide value return type, LAT (Lowest Astronomical Tide) '
+              'or MSL (Mean Sea Level)')
+@click.option('-p', '--mikepath', type=click.Path(dir_okay=True, file_okay=False),
+              default=r'C:\Program Files (x86)\DHI\2016', show_default=True,
+              help='Path to DHI MIKE version install root directory')
+def points(date, timestamp, **kwargs):
+    """Create a point shapefile containing tide values over an AOI
 
     Example use:
 
-    tidepods tides -p "C:/Program Files (x86)/DHI/2016/MIKE SDK/bin" -i C:/tides/aoi.dfs0
-    -d 20150131 -t 12:34 -s C:/tides/pts.shp -o C:/tides/pts_tides.shp
+    tidepods points -p "C:/Program Files (x86)/DHI/2016/MIKE SDK/bin" -i C:/tides/aoi.tif
+    -d 20150131 -t 10:30 -o C:/tides/pts_tides.shp
     """
     kwargs.update(date=dt.combine(date, timestamp))
-    from . import write_tides as wt
-    wt.write_tide_values(**kwargs)
+    from . import create_tides as ct
```

Comment:
We don't use the `from . import` notation if we can avoid it. Absolute imports are better: `from tidepods import`

Reply:
Sorted